doc/classes/int.xml

```diff
@@ -339,12 +420,29 @@
 			<argument index="0" name="right" type="int">
 			</argument>
 			<description>
+				Returns the result of bitwise [code]OR[/code] operation for two integers.
+				[codeblock]
+				print(2 | 4) # 6
+				print(1 | 3) # 3
+				[/codeblock]
+				It's useful to store binary flags in a variable.
+				[codeblock]
+				var flags = 0
+				# Turn first and third bit on.
+				flags |= 1
+				flags |= 4
+				[/codeblock]
 			</description>
 		</method>
 		<method name="operator ~" qualifiers="operator">
 			<return type="int">
 			</return>
 			<description>
+				Returns the result of bitwise [code]NOT[/code] operation for the integer. It's effectively equal to -[int] + 1.
```

Comment:
```suggestion
				Returns the result of bitwise [code]NOT[/code] operation for the integer. It's effectively equal to [code]-[int] + 1[/code].
```

Reply:
References don't work inside [code], so I made it an int.